lib/ssh_scan/policy.rb

```diff
@@ -25,5 +26,20 @@ def self.from_string(string)
       opts = YAML.load(string)
       self.new(opts)
     end
+
+    # Choose a policy file based on rules
+    def self.first_match(result)
+      Dir.glob("policies/**/*.yml") do |policy_file|
```

Comment:
This might not work once we release the code as a gem, consider expanding the path based on current location of this file.